utils/a11y_controller.ts

```diff
@@ -834,14 +849,22 @@ export default class A11yController {
         this.mouseIsPressed = false;
     }
 
-    handleFocus = (event) => {
+    handleFocus = (event: Event) => {
+        if (
+            !(event instanceof FocusEvent) ||
+            !event.target ||
+            !(event.target instanceof HTMLElement)
+        ) {
+            return;
+        }
+
         // since the post-list (in which the EditPost component lives) has the attribute `data-a11y-child-focus` hitting
         // the UP-Key causes the a11y controller to go through the posts. This is unwanted behavior in a textarea, so we
         // decided to leave this fix in for now. If we find the need for a more sustainable fix we can certainly do
         // that, as well, but for now this is sufficient.
         // @see: https://github.com/mattermost/mattermost-webapp/pull/8882#discussion_r790905592
         if (this.lastInputEventIsKeyDown && this.windowIsFocused && event.target.id !== 'edit_textbox') {
-            this.nextElement(event.target, event.path || true);
+            this.nextElement(event.target, event.composedPath() || true);
```

Comment:
I hope this change doesn't cause any problems. From what I can tell, `event.path` either never existed or was some poorly-documented, browser-specific behaviour since I could find very little information about it.

https://stackoverflow.com/a/30021867